static/css/inbox.less

```diff
@@ -762,6 +762,9 @@ a.fa:hover {
       }
 
       label {
+        &:disabled {
+          display: none;
+        }
```

Comment:
I think this should apply to all enketo forms and nowhere else, so I would add this to `static/css/enketo/medic.less` instead so it's wrapped in a `.enketo` selector.

Reply:
Upon closer inspection I see where it was added and you're right this feels hacky... can you try reducing the specificity of the `display: block` so it doesn't override the enketo styles?